proxy/api/protocol.go

```diff
@@ -28,6 +29,18 @@ type header struct {
 	flags  uint32
 }
 
+const maxPayloadLength = 1 * 1024 * 1024
```

Comment:
The shim uses a different limit on the payload, namely the one specified by hyperstart.
https://github.com/01org/cc-oci-runtime/blob/master/shim/shim.h#L61.
We should be consistent here.